Jenkinsfile-candidate

```diff
@@ -0,0 +1,34 @@
+#!groovy
+node {
+
+    load "$JENKINS_HOME/jobvars.env"
+
+    env.JAVA_HOME = "${tool 'openjdk-11'}"
+    env.PATH = "${env.JAVA_HOME}/bin:${env.PATH}"
+
+    stage('Checkout') {
+        checkout scm
+    }
+    stage('Assemble') {
+        sh "./gradlew clean assemble -P buildNumber=${env.BUILD_NUMBER}"
```

Comment:
will it execute the tests and build?

Reply:
Nope, test and build steps will be executed after that